services/inc/src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,5 @@
+info-service.url=http://localhost:8000
+basket-service.url=http://localhost:8010
+springfox.host=localhost:8012
```

Comment:
Do we have a service with port 8011?

Reply:
Why 8011? Because there is no port 8011 in this list?